packages/expo-image-picker/ios/ImagePickerOptions.swift

```diff
@@ -18,7 +17,7 @@ internal struct ImagePickerOptions: Record {
   var aspect: [Double]
 
   @Field
-  var quality: Double?
+  var quality: Double = 1.0
```

Comment:
This is a big change, but I think this is what people usually expect – to keep the original image.